downstream/modules/platform/proc-aap-platform-gateway-restore.adoc

```diff
@@ -1,3 +1,31 @@
-[id="aap-platform-gateway-restore"]
+[id="aap-platform-gateway-restore_{context}"]
 
-= Recovering your AnsibleAutomationPlatform resource
+= Recovering your AnsibleAutomationPlatform deployment
+The *AnsibleAutomationPlatform* hosts any enabled components (such as, {ControllerName}, {HubName}, {EDAName}, and {LightspeedShortName}), when you recover *AnsibleAutomationPlatform* you also restore these components.
+
+.Prerequisites
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been installed to the cluster.
+* The *AnsibleAutomationPlatformBackups* deployment is available in your cluster.
+
+.Procedure 
+. Log in to {OCP}.
+. Go to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Go to your btn:[All Instances] tab, and click btn:[Create New].
+. Select *Ansible Automation Platform Restore* from the list.
+. For *Name* enter the name for the recovery deployment. 
+. For *New Ansible Automation Platform Name* enter the new name for your {PlatformNameShort} instance. 
+. *Backup Source* defaults to *CR*.
+. For *Backup name* enter the name your chose when creating the backup. 
+. Click btn:[Create].
+
+Your backups will start restoring under the *AnsibleAutomationPlatformRestores* tab.
+
+NOTE: The recovery is not complete until all the resources are successfully restored. Depending on the size of your database this typically takes between 10-30 minutes.
```

Comment:
I wouldn't give a definite time unless this is guaranteed, some uses might have slow systems and they might worry if nothing happens after 10-30 mins. Better to say 'this can take some time.'